packages/metadata-editor/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@elyra/metadata-editor-extension",
+  "version": "1.0.0-dev",
+  "description": "JupyterLab extension - Edit metadata",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/elyra-ai/elyra",
+  "bugs": {
+    "url": "https://github.com/elyra-ai/elyra/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elyra-ai/elyra/"
+  },
+  "license": "Apache-2.0",
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf lib",
+    "dist": "npm pack .",
+    "prepare": "npm run build",
+    "watch": "tsc -w"
+  },
+  "dependencies": {
+    "@elyra/application": "^1.0.0-dev",
+    "@elyra/ui-components": "^1.0.0-dev",
+    "@jupyterlab/application": "^2.0.2",
+    "@jupyterlab/apputils": "^2.0.2",
+    "@jupyterlab/codeeditor": "^2.0.2",
+    "@jupyterlab/docregistry": "^2.0.2",
+    "@jupyterlab/fileeditor": "^2.0.2",
+    "@jupyterlab/launcher": "^2.0.2",
+    "@jupyterlab/logconsole": "^2.0.2",
+    "@jupyterlab/mainmenu": "^2.0.2",
+    "@jupyterlab/outputarea": "^2.0.2",
+    "@jupyterlab/ui-components": "^2.0.2",
+    "@jupyterlab/settingregistry": "^2.0.1",
+    "@jupyterlab/toc": "^4.0.0",
+    "@lumino/messaging": "^1.3.3",
+    "@lumino/widgets": "^1.11.1"
```

Comment:
do we need all these dependencies here only the ones imported in metadata-editor/src files?